app/lib/pre_assembly/batch.rb

```diff
@@ -50,10 +50,10 @@ def object_filenames_unique?(dobj)
     # @return [Enumerable<PreAssembly::DigitalObject>]
     # @yield [PreAssembly::DigitalObject]
     # rubocop:disable Metrics/AbcSize
-    def digital_objects(&_block)
+    def digital_objects
       return enum_for(:digital_objects) { containers_via_manifest.size } unless block_given?
 
-      containers_via_manifest.each_with_index.map do |container, i|
+      containers_via_manifest.each.with_index.map do |container, i|
```

Comment:
If you're open to it, I wonder if `containers_via_manifest.map.with_index` would work here. (I'd be a bit surprised to learn we need to chain an `each` with a `map`.)

Reply:
ah, yes, that makes sense and would read better, hadn't thought about that option.

Reply:
Why not `map.with_index`?

Reply:
@mjgiarlo & @jcoyne, i dropped the `each`, i think this is now what was being asked for (just `map.with_index`).

Reply:
Awesome, thanks!